<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kommivoyazher</class>
 <widget class="QMainWindow" name="Kommivoyazher">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kommivoyazher</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(4, 211, 159)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="kj">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
    <property name="text">
     <string>Длина:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jk">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>130</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
    <property name="text">
     <string>Маршрут:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_path">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="label_way">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>130</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
   </widget>
   <widget class="QLabel" name="graph">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>421</width>
      <height>361</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>60</y>
      <width>93</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 0, 0)</string>
    </property>
    <property name="text">
     <string>POWER</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(230, 230, 230)</string>
    </property>
    <property name="text">
     <string>Введите номер города:</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
